examples/jaeger-hotrod/docker-compose.yml

```diff
@@ -3,7 +3,7 @@ services:
   data-prepper:
     restart: unless-stopped
     container_name: data-prepper
-    image: opensearchproject/data-prepper:2
+    image: opensearchproject/data-prepper:2.5.0
```

Comment:
I tend to think that as this is a Data Prepper example, it should use the latest `2` version. We do try to ensure that we don't introduce breaking changes between versions. What do you think about keeping this one tied to `2`?

Reply:
Makes sense, done!